kubetest/e2e.go

```diff
@@ -28,6 +28,17 @@ import (
 	"time"
 )
 
+func argFields(args, dump, ipRange string) []string {
```

Comment:
Is this a temporary func or we are plan to expand it in the future? It seems specific to `dump` and `ipRange` now. Add a TODO or comment?